addons/sys_external/fnc_childrenActions.sqf

```diff
@@ -33,8 +33,8 @@ if (_playerOwnsRadio) then {
         QGVAR(giveRadio),
         _string,
         "",
-        {[(_this select 2) select 0, _target] call FUNC(stopUsingExternalRadio)},
-        {!([(_this select 2) select 0] call FUNC(isExternalRadioUsed))},
+        {[_radio, _target] call FUNC(stopUsingExternalRadio)},
```

Comment:
Where does this _radio come from? Action and condition code don't know about outer variables.

Reply:
Oh true! I was just testing and wondering :D

Reply:
There is no reason we don't just pass thr radio through instead of all params though, could just use `_this#2`/`_this select 2` and done.